### PR TITLE
Final cut michael revision

### PR DESCRIPTION
**Pull request by mich4xD**
Updating documentation for scripts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### What is it?
```
Comment (melissakendall) [inline README.md:3]: If you want developers to use this, I would strongly encourage you add to the top:
* Some kind of logo, people love logos
* If there is any form of testing/code here, people love seeing integrated test badges. If this is docs, that might not be relevant for this case

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
-Shoebox is an all-in-one bundle of tutorials and scripts (shell & docker-compose) for setting up a simple collaborative software development environment. It can be hosted on a VPS or dedicated server as an inexpensive alternative for subscription-based could services. Software components used in this setup are either open source or have free versions (some with limitations, please check).
+Shoebox is an all-in-one bundle of tutorials and scripts (shell & docker-compose) for setting up a simple collaborative software development environment. It can be hosted on a VPS or dedicated server as an inexpensive alternative for subscription-based cloud services. Software components used in this setup are either open source or have free versions (some with limitations, please check).
```
Comment (melissakendall) [inline README.md:5]: I will point you to [textlint](https://github.com/textlint/textlint), which is a tool that can prose-test markdown, you can even hook it up to github PRs and detect things like passive voice, etc.

**Comment by melissakendall** [comment]
I would recommend moving your Q&A to the bottom, that is where I would expect to find it. You can link to it from above

**Comment by bahram-aliyev** [comment]
Thanks, @melissakendall! Your comments were taken into account in #13.

**Pull request closed without merging**